.github/actions/checkout-helm-versions/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'checkout-helm-versions'
+description: 'Action to do a sparse checkout of environments and versions directories from https://github.com/broadinstitute/terra-helmfile/'
+author: 'ichengchang'
+inputs:
+  repository:
+    description: 'terra-helmfile repo'
+    required: true
+    default: 'broadinstitute/terra-helmfile'
```

Comment:
This action makes assumptions about the structure of the `helmfile` repository. Practically, will that ever work anywhere except the `broadinstitute/terra-helmfile` repo?

Reply:
You're correct this action practically makes assumptions on the file structure of `terra-helmfile` repo. Hence the specific action name `checkout-helm-versions`. The action could have been more general by passing in the exact directories to checkout as an input array but I decided against making the action more general than necessary at this stage. If someone found a more general use of the action in the future, it can easily be adapted. What do you think?

Reply:
I think it would make sense to parametrize this once we have at least two possible values. So for now I'd hardcode it. But I don't feel strongly either way, fine to leave as is if you prefer.

Reply:
I'll create a JIRA ticket for the parametrization. Thanks for reviewing. Merging the code now.